packages/deprecation-crawler/sandbox/deprecations/all-lowercase.md

```diff
@@ -1,9 +1,9 @@
 <!-- ruid-groups
 
 - master: 
-  - https://github.com/timdeschryver/find-deprecations/tree/master/all-lowercase/crawled.ts#L12
-  - https://github.com/timdeschryver/find-deprecations/tree/master/all-lowercase/crawled.ts#L18
-  - https://github.com/timdeschryver/find-deprecations/tree/master/all-lowercase/crawled.ts#L24
+  - /tree/master/all-lowercase/crawled.ts#L12
```

Comment:
This change is here because the git command in sandbox mode return an empty string